packages/arb-util/broadcastclient/broadcastclient.go

```diff
@@ -105,15 +123,63 @@ func (bc *BroadcastClient) ConnectInBackground(ctx context.Context, messageRecei
 	})()
 }
 
-func (bc *BroadcastClient) connect(ctx context.Context, messageReceiver chan broadcaster.BroadcastFeedMessage) (io.Reader, chan broadcaster.BroadcastFeedMessage, error) {
+var ErrIncorrectFeedServerVersion = errors.New("incorrect feed server version")
+var ErrIncorrectChainId = errors.New("incorrect chain id")
+
+func (bc *BroadcastClient) connect(ctx context.Context, messageReceiver chan broadcaster.BroadcastFeedMessage, previousSequenceNumber *big.Int) (io.Reader, chan broadcaster.BroadcastFeedMessage, error) {
 
 	if len(bc.websocketUrl) == 0 {
 		// Nothing to do
 		return nil, nil, nil
 	}
 
+	var requestedSequenceNumber string
+	if previousSequenceNumber.Cmp(big.NewInt(0)) > 0 {
+		requestedSequenceNumber = new(big.Int).Add(previousSequenceNumber, big.NewInt(2)).String()
```

Comment:
```// previousSequenceNumber is 1 before current, and we want 1 after current, so add 2.```

Reply:
It looks like `previousSequenceNumber` is equal to `bc.lastInboxSeqNum` in one place and ` mon.Core.GetMessageCount()` so in which case it's he current sequence numbered we should just add 1?

Reply:
`lastInboxSeqNum` is confusing.  For classic, feed messages contain the sequence number of the last message, not the current message.  I believe you got `mon.Core.GetMessageCount()` when `arb-node.go` calls `NewBroadcastClient`, but `NewBroadcastClient` subtracts one from that parameter so that it is the same as `lastInboxSeqNum`